extensions/amp-fx-flying-carpet/0.1/amp-fx-flying-carpet.css

```diff
@@ -44,9 +44,10 @@ amp-fx-flying-carpet > .-amp-fx-flying-carpet-clip {
 amp-fx-flying-carpet > .-amp-fx-flying-carpet-clip > .-amp-fx-flying-carpet-container {
   position: fixed !important;
   top: 0 !important;
+  width: 100%;
```

Comment:
Defaults to 100% width, so that `fixed-height` elements get a width during initial measure. This will be updated in the next mutate phase to match the flying-carpet's width exactly.